1.html

```diff
@@ -0,0 +1,243 @@
+        margin-right:13%;
+        
+        
+      }
+
+.byo{
+  background-color:#F6CA57;
+  border-radius:25px;
+  margin-left:34%;
+  width:100px;
+  font-size:52%;
+  text-align:center;
+
+}
+
+.sidepanel  {
+    width: 0;
+    position: fixed;
+    z-index: 1;
+    height: 250px;
+    top: 0;
+    left: 0;
+    background-color: #F6C041;
+    overflow-x: hidden;
+    transition: 0.5s;
+    padding-top: 60px;
+    border-radius:20px;
+  }
+  
+  .sidepanel a {
+    padding: 8px 8px 8px 32px;
+    text-decoration: none;
+    font-size: 25px;
+    color: #FFFFFF;
+    display: block;
+    transition: 0.3s;
+    font-family: 'Josefin Sans', sans-serif;
+  }
+  
+  .sidepanel a:hover {
+    color: #E26B2B;
+  }
+  
+  .sidepanel .closebtn {
+    position: absolute;
+    top: 0;
+    right: 25px;
+    font-size: 36px;
+  }
+  
+  .openbtn {
+    font-size: 20px;
+    cursor: pointer;
+   background-color: transparent;
+    color: white;
+    padding: 10px 15px;
+    border: none;
+    border-radius:10px;
+    float: left;
+  }
+  
+  .openbtn:hover {
+    background-color:#F6C041;
+  }
+
+#myBtn{
+    color:black;
+    border: none;
+    background-color: inherit;
+     padding: 14px 28px;
+  font-size: 16px;
+  cursor: pointer;
+  display: inline-block;
+    border-radius:35px;
+
+}
+
+
+#myBtn:hover {
+  color: white;
+  font-size:30px;
+    font-weight:bolder;
+    background-color:#F6CA57;
+}    
+
+#myBtn1{
+    color:black;
+    border: none;
+    background-color: inherit;
+     padding: 14px 28px;
+  font-size: 16px;
+  cursor: pointer;
+  display: inline-block;
+    border-radius:35px;
+}
+
+#myBtn1:hover {
+  background-color: #F6CA57;
+  color: white;
+  font-size:30px;
+    font-weight:bolder;
+}    
+
+#myBtn2{
+    color:black;
+    border: none;
+    background-color: inherit;
+     padding: 14px 28px;
+  font-size: 16px;
+  cursor: pointer;
+  display: inline-block;
+    border-radius:35px;
+}
+
+#myBtn2:hover {
+  background-color: #F6CA57;
+  color: white;
+  font-size:30px;
+    font-weight:bolder;
+}    
+
+#demo{
+    color:black;
+}
+      
+#myFinish{
+  color:black;
+    border: none;
+    background-color: white;
+     padding: 10px 15px;
+  font-size: 16px;
+  cursor: pointer;
+  display: inline-block;
+    border-radius:35px; 
+      }
+      
+     
+      #myFinish:hover {
+        background-color: #F6CA57;
+  color: white;
+  font-size:20px;
+    font-weight:bolder;
+      }
+ </style>
+
+<!-- This is the JS  -->
+
+<script>
+document.getElementById("myBtn").addEventListener("click", myFunction);
+function myFunction() {
+  document.getElementById("demo").innerHTML = "$9.95";
+}
+
+document.getElementById("myBtn1").addEventListener("click", myFunction1);
+function myFunction1() {
+  document.getElementById("demo").innerHTML = "$14.95";
+}
+
+document.getElementById("myBtn2").addEventListener("click", myFunction2);
+function myFunction2() {
+  document.getElementById("demo").innerHTML = "$19.95";
+}
```

Comment:
I wouldn't change this at this point - there's more important things for you to prioritize - but another way you could write this would be to have each of these buttons have the same class, and add the price as a data attribute in the HTML. Then, when you click any of the buttons, you could change the innerHTML of the demo to use the value in your data attribute instead of manually coding each. This would be a really great change to make if you have time after addressing other items here!